internal/sirius/create_additional_draft_test.go

```diff
@@ -0,0 +1,80 @@
+package sirius
+
+import (
+	"context"
+	"fmt"
+	"github.com/pact-foundation/pact-go/dsl"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"testing"
+)
+
+func TestCreateAdditionalDraft(t *testing.T) {
+	t.Parallel()
+
+	pact := newPact()
+	defer pact.Teardown()
+
+	testCases := []struct {
+		name                string
+		additionalDraftData AdditionalDraft
+		setup               func()
+		expectedResponse    map[string]string
+		expectedError       func(int) error
+	}{
+		{
+			name: "Minimal",
+			additionalDraftData: AdditionalDraft{
+				CaseType: []string{"personal-welfare"},
+			},
+			setup: func() {
+				pact.
+					AddInteraction().
+					Given("I am a Modernised LPA user").
```

Comment:
I think this needs to be:
```suggestion
					Given("A donor exists").
```
To ensure that someone with ID 188 is already present